config_helper.py

```diff
@@ -0,0 +1,300 @@
+    return font
+
+def _find_option_in_config(option, card_type=None, card_name=None):
+    if config.has_section(card_name) and config.has_option(card_name, option):
+        return card_name
+    if config.has_section(card_type) and config.has_option(card_type, option):
+        return card_type
+    if config.has_section(DEFAULT_SECTION) and config.has_option(DEFAULT_SECTION, option):
+        return DEFAULT_SECTION
+    return None
+
+def _get_resource(option, card_type=None, card_name=None):
+    '''
+    @param str option: The option to pull the value for
+    @param str card_type: If not None, will check this section for the given option.
+    @param str card_name: If not None, will check this section for the given option
+        before 'card_type'.
+
+    Attempts to return the value for the given option from the resources_cache dict,
+        first from a section matching card_name, then from a section matching
+        card_type, then the Card Defaults section. If none of the sections
+        (if they exist) have that option, then None is returned.
+    All resources should be made the correct type as they're loaded during LoadConfig,
+        so no checking of types is performed in this function.
+
+    @return str or None: None, or the value from the config
+    '''
+    if card_name in resources_cache and option in resources_cache[card_name]:
+        section = card_name
+    elif card_type in resources_cache and option in resources_cache[card_type]:
+        section = card_type
+    elif DEFAULT_SECTION in resources_cache and option in resources_cache[DEFAULT_SECTION]:
+        section = DEFAULT_SECTION
+    else:
+        return None
+    return resources_cache[section][option]
+
+def _get_list_from_config(option, card_type=None, card_name=None):
+    '''
+    Attempts to retrieve a value from the config file, then convert it to a
+    list of integers. If the conversion fails, it just returns the string value.
+    
+    The list must be separated by commas. Leading and trailing spaces around
+    the items in the list are allowed. Any leading or trailing spaces or
+    brackers ()[] will be stripped off the string before splitting around the
+    commas.
+
+    @return tuple
+    '''
+    section = _find_option_in_config(option, card_type, card_name)
+    if section is None:
+        raise ConfigParseError("Option not found in config: {}|{}|{}".format(
+            option, card_type, card_name
+            ))
+    value_str = get(option, section)
+    # Try to convert str to list
+    try:
+        return [int(item) for item in value_str.strip(' ()[]').split(',')]
+    except ValueError:
+        return value_str
+
+def _find_and_load_image(option, card_type=None, card_name=None,
+                        resource_directory=None):
+    '''
+    Checks for an image in the resources_cache dict. If it's not there,
+    checks the config file and attempts to load the image into the
+    resources_cache directory. 
+    '''
```

Comment:
Correct. You've reminded me that I'll need to make a separate get function for retrieving card art, something that knows in what directory the card art is stored. Probably getart() or something.